CONTRIBUTING.md

```diff
@@ -148,14 +148,30 @@ During a peer review, reviewers will add a tag to the comment to help the author
 [Kudos]|The reviewer liked this, specifically. Go you.
 [Suggestion]| An optional, alternative approach.
 
-
-
 Adding multiple tags to a comment is fine.  Do not combine tags.
 
 Good: ``[Suggestion][Kudos] This is great, but...``
 
 Bad:  ``[Suggestion|Kudos] This is great, but...``
 
+
+## Pull request labels
+Labels may be applied to your pull requests, for the purposes of organization and / or overall 
+status.
+
+| label | Explanation |
+| --- | --- |
+Awaiting changes    | this pull request is on hold until requested changes have been effected
+Feature             | this pull request adds a new feature
+Bug Fix             | this pull request fixes some bug
+Refactor            | this pull request refactors something
+Documentation       | this pull request only effects documentation
+Linting             | this pull request adjusts our linting configurations
+CI change           | This pull request reconfigures the CI environment
```

Comment:
Capitalization should be consistent throughout this table.
Either every cell has the first letter capitalized, or none do. (This applies to the `label` header, too.)

Reply:
resolved in 5aa1fd6